Dockerfile

```diff
@@ -18,9 +18,11 @@ RUN make build
 # Create a minimal container to run a Golang static binary
 FROM scratch
 
+COPY --from=tarampampam/curl /bin/curl /curl
 COPY --from=builder /usr/share/zoneinfo /usr/share/zoneinfo
 COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
 COPY --from=builder /go/whoami/whoami .
 
+HEALTHCHECK --interval=10s --start-period=2s CMD ["/curl", "--fail", "http://127.0.0.1:80/health"]
```

Comment:
Because of the increased CPU usage, I don't think we should add a `HEALTHCHECK` directive to the Dockerfile to enable the health check by default. Instead, we should add documentation on how the users could enable the health check themselves with the `--health-*` Docker CLI flags, or the `healthcheck:` Docker Compose attribute, only IF they want to.